functions_framework_builder/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: functions_framework_builder
-version: 0.2.0-dev
+version: 0.2.0
```

Comment:
Update the changelog to be `0.2.0` now, too!

Reply:
I shouldn't have switched from draft PR just yet, because I got this already. Just pushing commits one by one for each step of testing/validation.